### PR TITLE
add artifacthub.io

### PR DESCRIPTION
**Pull request by fharper**

### CODE REVIEW

File: img/a/artifacthub.io.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...15.715 0 0 1-6.526-1.366 15.059 15.059 0 0 1-4.856-3.946 17.989 17.989 0 0 1-3.186-6.373 33.756 33.756 0 0 1-1.063-8.65 31.489 31.489 0 0 1 1.063-8.347 20.519 20.519 0 0 1 3.035-6.373 13.016 13.016 0 0 1 5.007-4.098 18.45 18.45 0 0 1 6.981-1.366 14.19 14.19 0 0 1 5.008.607 24.21 24.21 0 0 1 3.49 1.518 23.927 23.927 0 0 1 2.428 1.517 3.183 3.183 0 0 0 1.67.607 1.635 1.635 0 0 0 1.214-.303 5.148 5.148 0 0 0 .91-.91l2.277-3.339a23.856 23.856 0 0 0-7.74-5.16 27.087 27.087 0 0 0-9.864-1.82A29.613 29.613 0 0 0 509.28 89.7a23.085 23.085 0 0 0-7.892 5.766 27.762 27.762 0 0 0-4.856 8.802 38.003 38.003 0 0 0 .152 22.763 24.834 24.834 0 0 0 5.008 8.65 20.232 20.232 0 0 0 7.436 5.312 23.974 23.974 0 0 0 9.257 1.82 30.456 30.456 0 0 0 11.078-1.972 19.152 19.152 0 0 0 8.802-6.222l-2.732-3.49a2.164 2.164 0 0 0-1.821-.91ZM647.072 96.074h-25.646V69.518h-23.37v72.69h23.37v-27.62h25.646v27.62h23.37v-72.69h-23.37v26.556zM735.241 112.616a14.303 14.303 0 0 1-3.338 9.712 14.37 14.37 0 0 1-19.121 0 14.303 14.303 0 0 1-3.34-9.712V69.517h-23.37v43.402c0 10.016 3.036 17.604 9.41 22.763 6.373 5.312 15.327 7.891 26.86 7.891s20.487-2.731 26.709-7.89 9.408-12.748 9.56-22.916V69.366h-23.37Z" class="cls-2"/></svg>
```
Comment (phallobst) [inline img/a/artifacthub.io.svg]: Here's an optimized icon without the text.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 177.8 195.4"><path fill="#fff" d="M163 41.2 101 5.8a24.2 24.2 0 0 0-24.2 0L14.7 41.3A24.2 24.2 0 0 0 2.6 62.2v71a24.4 24.4 0 0 0 12.1 21l62.1 35.4a24.2 24.2 0 0 0 24.2 0l62.1-35.5a24.3 24.3 0 0 0 12.1-20.9v-71a24.4 24.4 0 0 0-12.1-21zM83.3 178.5l-62-35.5a11.4 11.4 0 0 1-5.7-9.8v-71a11.3 11.3 0 0 1 5.6-9.8L83.3 17a11.3 11.3 0 0 1 5.6-1.5 10.8 10.8 0 0 1 5.7 1.4l62 35.5a11.4 11.4 0 0 1 5.7 9.8v71a11.3 11.3 0 0 1-5.6 9.8l-62.1 35.5a11.4 11.4 0 0 1-5.7 1.5 11.2 11.2 0 0 1-5.7-1.5z"/><path fill="#417598" d="M177.8 62.2A27 27 0 0 0 164.4 39L102.3 3.6a26.8 26.8 0 0 0-26.7 0L13.4 39A26.8 26.8 0 0 0 0 62.2v71a27 27 0 0 0 13.4 23.2l62.1 35.5a26.7 26.7 0 0 0 26.8 0l62.1-35.5a26.8 26.8 0 0 0 13.4-23.2zm-2.6 71a24.3 24.3 0 0 1-12 21L101 189.5a24.2 24.2 0 0 1-24.2 0l-62-35.5a24.4 24.4 0 0 1-12.2-21V62.3a24.2 24.2 0 0 1 12-21L76.9 5.9a24.2 24.2 0 0 1 24.2 0l62 35.4a24.4 24.4 0 0 1 12.2 21zm-18.5 9.8a11.3 11.3 0 0 0 5.6-9.8v-71a11.4 11.4 0 0 0-5.6-9.8L94.6 17a10.8 10.8 0 0 0-5.7-1.4 11.3 11.3 0 0 0-5.6 1.5L21 52.4a11.3 11.3 0 0 0-5.6 9.8v71a11.4 11.4 0 0 0 5.7 9.8l62 35.5A11.2 11.2 0 0 0 89 180a11.4 11.4 0 0 0 5.7-1.5z"/></svg>
```

**Review by phallobst — CHANGES_REQUESTED** [review]
Do you have a documentation link or screenshots of the setup process (without any personal information)?

File: img/a/artifacthub.io.svg (hunk 1 above)
Comment (fharper) [reply img/a/artifacthub.io.svg]: Oh, I used the tool proposed, thanks

**Comment by fharper** [comment]
I didn't find any documentation. I can deactivate 2fa & do it again, but there's always 2-3 steps and it's pretty standard.

**Review by phallobst — APPROVED** [review]

**Pull request merged**